client_plugin/drivers/shared/etcdops.go

```diff
@@ -134,7 +163,7 @@ func (d *VolumeDriver) initEtcd() error {
 					"manager ID": nodeID},
 			).Info("Swarm node role: manager. Action: find leader ")
 
-			joinEtcdCluster(addr, n.ManagerStatus.Addr, nodeID)
+			e.joinEtcdCluster(n.ManagerStatus.Addr)
 			return nil
```

Comment:
Thanks for catching this... I changed the logic of return last time and forgot to update return error here!